frontend/awx/access/users/Users.cy.tsx

```diff
@@ -0,0 +1,75 @@
+import { AwxUser } from '../../interfaces/User';
+import * as useOptions from '../../../common/crud/useOptions';
+import { Users } from './Users';
+import { ToolbarFilterType } from '../../../../framework';
+
+describe('Users.cy.ts', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: 'GET',
+        url: '/api/v2/users/*',
+        hostname: 'localhost',
+      },
+      {
+        fixture: 'users.json',
+      }
+    ).as('getUsers');
+    cy.intercept(
+      {
+        method: 'OPTIONS',
+        url: '/api/v2/users/',
+      },
+      {
+        fixture: 'mock_options.json',
+      }
+    ).as('getOptions');
+  });
+
+  it('renders users list', () => {
+    cy.mount(<Users />);
+    cy.verifyPageTitle('Users');
+    cy.get('table').find('tr').should('have.length', 10);
+  });
+
+  it('Create User button is disabled if the user does not have permission to create users', () => {
+    cy.intercept({ method: 'GET', url: '/api/v2/users/*' }, { fixture: 'users.json' });
+    cy.mount(<Users />);
+    cy.contains('a', /^Create user$/).should('have.attr', 'aria-disabled', 'true');
+  });
+
+  it('create User button is enabled if user has permissions', () => {
+    cy.stub(useOptions, 'useOptions').callsFake(() => ({
+      data: {
+        actions: {
+          POST: {
+            name: {
+              type: ToolbarFilterType.MultiText,
+              required: true,
+              label: 'Name',
+              max_length: 512,
+              help_text: 'Name of this user.',
+              filterable: true,
+            },
+          },
+        },
+      },
```

Comment:
It is unclear to me what this stub does/affects user permissions. Why are we importing `ToolbarFilterType` here?

Reply:
That stub is used to ensure the user is able to create users and that the create user button is enabled. I can play around with figuring out if importing ToolbarFilterType is necessary anymore